benchmarks/test_distances.py

```diff
@@ -44,7 +44,7 @@
 def Neff_from_N_ur_N_ncdm(N_ur, N_ncdm):
     """Calculate N_eff from the number of relativistic
     and massive neutrinos."""
-    Neff = N_ur + N_ncdm * ccl.cosmology._Defaults.T_ncdm**4 / (4/11)**(4/3)
+    Neff = N_ur + N_ncdm * ccl.DefaultParams.T_ncdm**4 / (4/11)**(4/3)
```

Comment:
All the parameters were public except that one. So I made it public and gave it a sensible name, just in case users want to see what the module-level default params are.